api/DedicatedWorkerGlobalScope.json

```
@@ -85,7 +85,7 @@
               "version_added": true
             },
             "webview_android": {
-              "version_added": true
+              "version_added": "37"
```

I couldn't find out if this existed earlier or not

```suggestion
              "version_added": "≤37"
```